### PR TITLE
feat: Add static file to ingress

### PR DESCRIPTION
**Pull request by lemaitre-aneo**
Add the support to serve a static json file to give information about the environment.

### CODE REVIEW

**Comment by esoubiran-aneo** [comment]
Are u sure that the target branch must be `release`?

File: armonik/variables.tf [hunk 1]
```diff
@@ -355,3 +355,9 @@ variable "metrics_server_chart_name" {
   type        = string
   default     = "metrics-server"
 }
+
+variable "environment_description" {
```
Comment (esoubiran-aneo) [inline armonik/variables.tf:359]: My knowledge of terraform is limited but when I will deploy ArmoniK in test, is there a way to change these value?

Comment (lemaitre-aneo) [reply armonik/variables.tf:359]: yes, in the `parameters.tfvars`. You can see an exemple in https://github.com/aneoconsulting/ArmoniK/pull/1161

**Comment by lemaitre-aneo** [comment]
> Are u sure that the target branch must be `release`?

As it is not main, I don't know much what should be the target.

@lzianekhodja ?

File: armonik/variables.tf (hunk 1 above)
Comment (lemaitre-aneo) [reply armonik/variables.tf:359]: > And does the normalization indicated in this issue,[aneoconsulting/ArmoniK#1148](https://github.com/aneoconsulting/ArmoniK/issues/1148), will be in this infra or only in the documentation?

For now, no normalization is applied, but if any, I think it should be directly in ArmoniK

**Review by esoubiran-aneo — APPROVED** [review]
LGTM 👍 Nice work!

**Review by lzianekhodja — APPROVED** [review]

**Comment by lzianekhodja** [comment]
@lemaitre-aneo we will do some clean, update examples in ArmoniK repository and we create a new release for ArmoniK.Infra.

**Pull request merged**